ru_RU/webapp/src/main/webapp/i18n/all_ru_RU.properties

```diff
@@ -0,0 +1,965 @@
+sub_properties = Подсвойство
+subproperty = подсвойство
+
+manage_labels_for = Управлять метками
+manage_labels_capitalized = Управление метками
+manage_labels_intro = В случае, если на одном языке существует несколько меток, воспользуйтесь ссылкой для удаления, чтобы удалить метки, которые вы не хотите отображать на странице профиля для данного языка. 
+processing_icon = в обработке
+selection_in_process = Идёт обработка выбранной вами информации.
+view_labels_capitalized = Просмотр меток
+view_labels_for = Просмотреть метки для
+
+select_an_existing_document = Выбрать существующий материал
+
+datetime_year_required = Интервалы времени должны начинаться с года. Введите либо год начала, либо год окончания, либо оба.
+select_a_language = Выберите язык
+
+base_property_capitalized = Базовое свойство
+faux_property_capitalized = Номинальное свойство
+faux_property_listing = Список номинальных свойств
+faux_property_by_base = номинальные свойства, упорядоченные по базовому свойству
+faux_property_alpha = номинальные свойства в алфавитном порядке
+
+no_class_restrictions = Не найдено классов с ограничением на задание данного отношения.
+
+invalid_format = Недопустимый формат
+four_digit_year = Неверный ввод. Пожалуйста, введите год как четырёхзначное число.
+year_numeric = Неверный ввод. Год должен быть вводится как число.
+year_month_day = Неверный ввод. Пожалуйста, введите год, месяц и день.
+minimum_ymd = Неверный ввод. Пожалуйста, введите хотя бы год, месяц и день.
+minimum_hour = Неверный ввод. Пожалуйста, укажите хотя бы час.
+year_month = Неверный ввод. Пожалуйста, введите год и месяц.
+decimal_only = Неверный ввод.  Десятичная точка разрешена, а разделители тысяч - нет.
+whole_number = Неверный ввод. Введите целое число без десятичной точки или разделителя тысяч.
+
+#
+# get label for class ( VClassDaoJena.java )
+#
+restriction_on = ограничения на
+all_values_from = все значения из
+some_values_from = некоторые значения из
+has_value = имеет значение
+minimum_cardinality = наименьшая значимость
+maximum_cardinality = наибольшая значимость
+cardinality = значимость
+not = не
+and = и
+or = или
+
+#
+# validation messages ( BasicValidationVTwo.java )
+#
+
+required_field_empty_msg = Это поле не может быть оставлено пустым.
+data_not_past_msg = Пожалуйста, укажите будущую дату публикации (прошедшие даты недопустимы).
+invalid_date_form_msg = должна быть в допустимом формате даты ММ/ДД/ГГГГ.
+file_must_be_entered_msg = в этом поле должно быть иимя файла.
+invalid_url_msg = Этот URL должен начинаться с http:// или https://
+
+label.dateTimeWithPrecision.year_capitalized = год
+label.dateTimeWithPrecision.month_capitalized = Месяц
+label.dateTimeWithPrecision.day_capitalized = День
+label.dateTimeWithPrecision.hour_capitalized = Часы
+label.dateTimeWithPrecision.minutes_capitalized = Минуты
+label.dateTimeWithPrecision.seconds_capitalized = Секунды
+confirm_individual_deletion = Вы уверены, что хотите удалить следующий объект? 
```

Comment:
not yet merged (pull request which adds this property), but hopefully will be soon

Reply:
Removed